setup.py

```diff
@@ -18,15 +18,15 @@
 from oscar import get_version  # noqa isort:skip
 
 install_requires = [
-    'django>=2.2,<3.3',
+    'django>=2.2,<3.2',
     # PIL is required for image fields, Pillow is the "friendly" PIL fork
     'pillow>=6.0',
     # We use the ModelFormSetView from django-extra-views for the basket page
     'django-extra-views>=0.13,<0.14',
     # Search support
     'django-haystack>=3.0b1',
     # Treebeard is used for categories
-    'django-treebeard>=4.3,<4.5',
+    'django-treebeard>=4.3',
```

Comment:
This was pinned temporarily because of a breaking change in treebeard, which has since been fixed.